Automattic/jetpack
# CRM: move quote delete away from update button

This PR moves the delete button further away from the update button.

## Proposed changes:



### Other information:


## Jetpack product discussion


https://github.com/Automattic/zero-bs-crm/issues/2947

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the edit page for a quote (e.g. `wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=quote&zbsid=1`)
* Check if the new placement of the Delete button looks like the after picture.

Before:
![image](https://user-images.githubusercontent.com/37049295/226788387-8a925e8e-bdc0-4156-9b82-9c7960ce90e1.png)

After:
![image](https://user-images.githubusercontent.com/37049295/226788534-18e45733-b780-4ff2-ac9f-86eecb3901e4.png)
